github-actions/src/main/scala/org/typelevel/sbt/gha/WorkflowJob.scala

```diff
@@ -149,6 +152,9 @@ object WorkflowJob {
     override def withEnvironment(environment: Option[JobEnvironment]): WorkflowJob = copy(environment = environment)
     override def withConcurrency(concurrency: Option[Concurrency]): WorkflowJob = copy(concurrency = concurrency)
     override def withTimeoutMinutes(timeoutMinutes: Option[Int]): WorkflowJob = copy(timeoutMinutes = timeoutMinutes)
+
+    def updatedEnv(name: String, value: String): WorkflowJob = withEnv(env.updated(name, value))
+    def appendSteps(steps: List[WorkflowStep]): WorkflowJob = withSteps(this.steps ++ steps)
```

Comment:
The method name `append` is inconsistent (in both name and signature) with `appended`. https://www.scala-lang.org/api/current/scala/collection/immutable/List.html#appended-957

Reply:
changed `append` to `appended` and made it take a single step.  Added `concatStep` with a `TraversableOnce` for added flexibility.

Reply:
also changed `add*` to `concat*` for consistency.